PrizeDraw/ViewModels/TileViewModel.cs

```diff
@@ -14,6 +14,20 @@ public class TileViewModel : ViewModelBase
         public SolidColorBrush Color { get; }
         public string RemoteImageUri;
         public BitmapImage BitmapImage { get; private set; }
+        private bool _isDrawn = false;
+        private bool _isWinner = false;
+
+        public bool IsWinner
+        {
+            get => _isWinner;
+            set
+            {
+                _isWinner = value;
+                _isDrawn = true;
```

Comment:
Why not? Surely it's implicit. If you've won the draw then you've been drawn. Don't get me wrong, I'm not precious about it but you can't win without being drawn so it seemed logical to set them at the same time to me.

Reply:
It's quite dangerous code and could well introduce future bugs. Imagine requirements change, and someone's working on some code, and writes the following...

    tile.IsDrawn = false;
    tile.IsWinner = true;

Those two lines are very simple, and one would expect the value of `IsDrawn` to be false. But with your change, `IsDrawn` would be _magically_ set to true. As a user of these properties, I wouldn't expect setting a simple properly called `IsWinner` to be changing other properties behind the scenes. To know that this was happening, I'd have had to have viewed the definition of that property.

Hidden side effects are almost always a bad thing - both properties and methods should do what they say on the tin, and try to avoid hidden other stateful side effects.

Reply:
In your new commit, can't you just use auto-properties? Ie. you've currently got all this...

![image](https://user-images.githubusercontent.com/567988/68081797-6f7bf200-fe0b-11e9-83e2-4cf2bfeef42a.png)

Rather than just:

    public bool IsWinner { get; set; }
    public bool IsDrawn { get; set; }